solidity/contracts/stubs/RewardsStub.sol

```diff
@@ -28,10 +28,28 @@ contract RewardsStub is Rewards {
         receiveReward(bytes32(i));
     }
 
+    function receiveRewards(uint256[] memory identifieres) public {
+        uint256 len = identifieres.length;
+        bytes32[] memory bytes32identifiers = new bytes32[](len);
+        for (uint256 i = 0; i < identifieres.length; i++) {
+            bytes32identifiers[i] = bytes32(identifieres[i]);
+        }
+        receiveRewards(bytes32identifiers);
+    }
+
     function reportTermination(uint256 i) public {
         reportTermination(bytes32(i));
     }
 
+    function reportMultipleTerminations(uint256[] memory identifieres) public {
```

Comment:
Do we need to have `*multiple*` in the name of this function? It looks it's not required for `receiveRewards`.

Reply:
Truffle didn't catch that I wanted to call a method from Stub and that's why I added 'multiple'. Thanks for the hint that we should call it differently in test! Renamed it.